pkg/master/ports/ports.go

```diff
@@ -17,7 +17,13 @@ limitations under the License.
 package ports
 
 const (
-	// KubeletPort is the default port for the kubelet status server on each host machine.
+	// KubeletStatusPort is the default port for the kubelet healthz server.
+	// May be overridden by a flag at startup.
+	KubeletStatusPort = 10248
```

Comment:
I was confused about the port definitions here but tried to follow the existing convention. 

ProxyPort (which I renamed to ProxyStatusPort because it's only used for the healthz server) isn't actually referenced anywhere in the code base. Neither is KubeletStatusPort (which I added here). I added it to make the port reservation clear but didn't introduce a package dependency from the kubelet to the master. 

A reasonable alternative would be to rip out these unused variables.